dev/com.ibm.ws.security.audit.source.rest/src/com/ibm/ws/security/audit/source/rest/MemberManagementEvent.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.security.audit.event;
+package com.ibm.ws.security.audit.source.rest;
```

Comment:
Looks like these two classes can just move over to the com.ibm.ws.request.probe.audit.rest project.  Looks like it is the only user of these classes.

Reply:
Should we keep them here to make them consistent the other events?